CoreFoundation/String.subproj/CFString.h

```diff
@@ -165,7 +165,7 @@ struct __CFConstStr {
         uint8_t _pad[4];
     } _base;
     uint8_t *_ptr;
-    uint32_t _length;
+    signed long _length;
```

Comment:
I meant to change it to be the same as the length field in __CFString struct which is of type CFIndex.   CFIndex is signed long.